tests/functional/glusterd/test_gluster_does_not_do_posix_lock_when_multiple_client.py

```
@@ -0,0 +1,78 @@
+"""
+Copyright (C) 2020 Red Hat, Inc. <http://www.redhat.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+Description:
+This test checks if gluster does not do posix lock
+when multiple clients are present.
+"""
+
+from tests.nd_parent_test import NdParentTest
+
+# nonDisruptive;dist,rep,dist-rep,disp,dist-disp,arb,dist-arb
+
+
+class TestCase(NdParentTest):
+
+    def run_test(self, redant):
+        """
+        Steps:
+        1. Create all types of volumes.
+        2. Mount the brick on two client mounts
+        3. Prepare same script to do flock on the two nodes
+         while running this script it should not hang
+        4. Wait till 300 iteration on both the node
+        """
+        # NOTE: This test case should be run with 2 clients
```

A check condition here, like
```
if len(self.client_list) < 2:
    raise Exception("Insufficient client systems")
```

---

@nik-redhat let's add this check and then maybe we can take this change in. We can run it in nightly...let it fail. Anyway..that's an env issue rather than framework or gluster issue.

---

Ack